bouquet/config.py

```
@@ -10,7 +10,7 @@
 # CalculatorFactory.available_methods() (the installed subset).
 from bouquet.calculator import MethodType
 
-__all__ = ["Configuration", "MethodType"]
+__all__ = ["Configuration", "RunOptions", "MethodType"]
```

Ruff (0.15.20)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)